docs/guides/configure-error-suppression.mdx

```diff
@@ -126,11 +130,11 @@ psi1_H1 = job.result()
 
 
 <span id="transpilation-table"></span>
-## Advanced runtime compilation options
+## Advanced error suppression options
 
 <Tabs>
   <TabItem value="PrimV2" label="V2 primitives">
-    In the V2 primitives, you can explicitly enable and disable individual error mitigation/suppression methods, such as dynamical decoupling.
+    In the V2 primitives, you can explicitly enable and disable individual error mitigation and suppression methods, such as dynamical decoupling. See [available options](runtime-options-overview#options-table) table for a list of available options.
 
     <Admonition type = "note">
     Dynamical decoupling is not supported when the input circuits are dynamic.
```

Comment:
Similar to the error mit page, it's better to point the compatibility table instead of singling out just DD.